dataland-frontend/src/components/pages/SingleDataRequest.vue

```diff
@@ -151,8 +151,6 @@ import { ApiClientProvider } from "@/services/ApiClients";
 import { assertDefined } from "@/utils/TypeScriptUtils";
 import ToggleChipFormInputs from "@/components/general/ToggleChipFormInputs.vue";
 import BasicFormSection from "@/components/general/BasicFormSection.vue";
-import { humanizeStringOrNumber } from "@/utils/StringFormatter";
-import { ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE } from "@/utils/Constants";
```

Comment:
what are these changes for? the framework dropdown on the single data request page is broken.

Reply:
@fsoffel can you have a look at it please?

Reply:
I assume this got here through some weird merge issus. However the lost imports are needed and I readded them.